app/assets/javascripts/components/event_form.js.jsx

```diff
@@ -0,0 +1,59 @@
+var ScheduleEventForm = React.createClass({
+  getInitialState: function () {
+    return {
+      scheduled: false
+    };
+  },
+
+  handleClick: function (event) {
+      $.ajax({
+        url: '/events',
+        method: "POST",
+        dataType: "JSON",
+        data: {
+          event: {
+            user_id: this.props.userID,
```

Comment:
This can be removed entirely and have the controller handle assigning it to the current user.